src/components/Banner/BannerContextManager.js

```diff
@@ -20,19 +24,82 @@ function addTimes(date, days) {
 }
 
 // Example usage:
-// activeBanner = {
-//   storageId: 'react_banner_XX',
-//   normalHeight: 60,
-//   smallHeight: 80,
-//   campaignStartDate: '2020-09-20Z', // the Z is for UTC
-//   campaignEndDate: '2020-10-31Z', // the Z is for UTC
-//   snoozeForDays: 7,
-//   content: dismiss => (
-//     <div>
-//       <a href="test">Test</a> <button onClick={dismiss}>close</button>
-//     </div>
-//   ),
-// };
+activeBanner = {
+  storageId: 'reactjs_banner_2020survey',
+  normalHeight: 100,
+  smallHeight: 140,
+  campaignStartDate: '2020-09-30Z', // the Z is for UTC
+  campaignEndDate: '2020-12-13Z', // the Z is for UTC
+  snoozeForDays: 7,
+  content: dismiss => (
+    <div
+      css={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'flex-start',
+      }}>
+      <a
+        href="https://www.surveymonkey.co.uk/r/673TZ7T"
```

Comment:
Should this open in new tab by default? Seems like you'd almost never want to go to the docs and then _go away_ to the survey.

Reply:
Do we have a policy of opening things in new tabs? I tend to err on letting the user right click and open in a new tab if they like. Whereas forcing opening in a new tab gives them no choice. Whatever works for you, though

Reply:
Yeah we generally have external links on the homepage open in a new tab.